### DISCUSSION TITLE
PV are not backed up or restored, if pod is not running.

### DISCUSSION THREAD

**Discussion by sebalaterra** [vmware-tanzu/velero#6963] [Community support Q&A]
Hello, 
We are using Velero within an EKS cluster with the node-agent daemonset to be able to back up data in volumes as well. 
We encountered a problem: actually volumes, attached to specific pods, are not backed up because the number of replicas (into the deployment) is set to 0 (so when the pod is not in a Running state). 
We wondered if this behavior is expected and if there is a way to back up the volume even if the pod is not running. 

Another problem we found in the restore phase, based on our experience it seems that volumes are not properly restored if the Pod where the volume is attached is not in a Running state. In our case the restored Pod was in a CrashLoopBackOff state and the volumes were not restored properly. 
I was wondering if there was a correlation between the status of the Pod and the success/failure of the Velero backup/restore. 
Thank you, 
Sebastiano

**Comment by sseago** [comment]
If you are using fs backup, the volumes must be mounted because the only way the node agent pod can access the volumes is via mount propagation which requires another pod on the same node to have the volume mounted.

If you need to back up volumes that are not mounted, snapshot backup doesn't have the same requirement.

It sounds like your restore problem is similar. If the volume-mounting pod you are restoring is not runnable, then if you're using fs backup, the node agent won't have access to those volumes.
